EventsExpress/ClientApp/src/components/profile/editProfile/editGender.js

```diff
@@ -3,9 +3,10 @@ import { Field, reduxForm } from "redux-form";
 import Button from "@material-ui/core/Button";
 import { renderSelectField } from '../../helpers/form-helpers'
 import ErrorMessages from '../../shared/errorMessage';
+import IconButton from "@material-ui/core/IconButton";
 
 let EditGender = props => {
```

Comment:
Is it nor possible to refactor edit gender?

Reply:
This component is not used, remove it?

Reply:
Maybe I can delete all the files Edit...?

Reply:
Yes, remove unused code please.